languages.yaml

```diff
@@ -23,6 +23,31 @@ en:
     BUILTIN_CSS: Use built in CSS
     BUILTIN_CSS_HELP: Include the CSS provided by the OAuth plugin
 
+dn:
```

Comment:
Can you update your commit to use `de:` rather than `dn:`? Our German short is `de`